js/Wallop.js

```diff
@@ -54,11 +54,14 @@
     this.createCustomEvent();
 
     // If there is no active item, start at 0
-    if (this.currentItemIndex < 0) {
+    if (this.currentItemIndex === -1) {
       this.currentItemIndex = 0;
       addClass(this.allItemsArray[this.currentItemIndex], this.options.currentItemClass);
     }
 
+    // Update button states to make sure the correct state is set on initialization
+    this.updateButtonStates();
```

Comment:
This is needed to set the correct button state on initialization when the `carousel` option is disabled. This was not the case before (see http://codepen.io/anon/pen/zvrWpp). To get the correct state for the previous button set, you need to click next and the previous.